scrap_flipkart.py

```diff
@@ -4,37 +4,52 @@
 
 
 def first_method(product_details):
-
-    for product_detail in product_details:
-
-        data = product_detail.findAll("div", {"class":"col col-7-12"})
-
-        for datum in data:
-            name = datum.div
-            print(name.text)  # PRINTING NAME OF EACH PRODUCT FOUND
-
-            rating = datum.find("div", {"class":"hGSR34"})
-            print(rating.text)  # PRINTING RATING OF EACH PRODUCT FOUND
-
-            detail = datum.findAll("li", {"class":"tVe95H"})
-            for k in detail:
+    for i in productDetail:

```

Comment:
You might want to use `product_details` instead of a silent global variable call

Reply:
Accepted